draft-ietf-quic-transport.md

```diff
@@ -468,7 +468,10 @@ Connection ID Flag:
 
 : The second bit (0x40) of the first octet indicates whether the Connection ID
   field is present.  If set to 1, then the Connection ID field is present; if
-  set to 0, the Connection ID field is omitted.
+  set to 0, the Connection ID field is omitted.  The Connection ID field can
+  only be omitted if the omit_connection_id transport parameter
+  ({{transport-parameter-definitions}}) is used by the intended recipient of the
```

Comment:
This isn't really a negotiation.  Each endpoint indicates whether it will accept a packet that omits connection ID.  Nothing that its peer says will change that.  "Used" is perhaps a little too generic, but would "includes" work?